about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./about.css">
+    <script src="https://kit.fontawesome.com/d9a272ac0d.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;300;400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+    <title>Confrence</title>
+</head>
+<body id="body">
+    <section class="header">
+        <header class="head">
+            <nav class="nav-bar">
+                <a id="logo" href="#"><i class="fa-sharp fa-solid fa-bars menu"></i></a>
+                <div class="mobile-menu">
+                  <div class="close-menu"><button class="menu-button"><img src="./images/close.png" alt="close-button" ></button></div>
+                  <ul class="mobnav">
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="./index.html">home</a></li><hr>
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="#">program</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Join</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Sponser</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">News</a></li><hr>
+                    </ul>
```

Comment:
- [ ] Kindly 😃 try to fix that in the nav of the header because when I click on the link to redirect me to the about page it is redirecting me, but the about link is not replaced with home to redirect me to the home again, so please fix that to make it like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

 | Your design| 
| --- | 
|  ![Recording 2023-03-17 at 13 39 56](https://user-images.githubusercontent.com/83360636/225894691-30757b7a-d316-45db-a1e6-c29cbbeef19d.gif)|